core/templates/domain/feedback_thread/FeedbackThreadObjectFactory.ts

```diff
@@ -21,9 +21,9 @@ import { downgradeInjectable } from '@angular/upgrade/static';
 import { Injectable } from '@angular/core';
 
 import { ThreadMessage } from
-  'domain/feedback_message/ThreadMessageObjectFactory';
-import { ThreadMessageSummary, ThreadMessageSummaryObjectFactory } from
-  'domain/feedback_message/ThreadMessageSummaryObjectFactory';
+  'domain/feedback_message/ThreadMessage.model';
```

Comment:
Move this to the above line (ditto below)

Reply:
done